test/preprocessors/MongoidPreprocessor.test.ts

```diff
@@ -67,8 +85,14 @@ describe("MongoidPreprocessor", () => {
     expect(entities[0].collectionName).toBeNull();
     expect(entities[1].collectionName).toBe("fhir_measures");
 
+    expect(entities[2].dataType.typeName).toBe("Resource");
+    expect(entities[2].memberVariables[3].variableName).toBe("fhirId");
+
+    expect(entities[3].dataType.typeName).toBe("Element");
+    expect(entities[3].memberVariables[2].variableName).toBe("fhirId");
+
     // Third entity is our hand-made FHIR.Type
-    expect(entities[2].dataType.namespace).toBe("FHIR");
-    expect(entities[2].dataType.typeName).toBe("Type");
+    expect(entities[4].dataType.namespace).toBe("FHIR");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)